### PR TITLE
man: fixes for fi_msg(3)

### PR DESCRIPTION
**Pull request by goodell**
Signed-off-by: Dave Goodell <dgoodell@cisco.com>

found during discussion prep for #598

### CODE REVIEW

File: man/fi_msg.3.md [hunk 1]
```diff
@@ -138,12 +138,12 @@ per call through the use of flags.  The fi_sendmsg function takes a
...
-	const struct iovec *msg_iov;/* scatter-gather array */
-	void               **desc;  /* local request descriptor */
-	size_t             iov_count;/* # elements in iov */
-	const void         *addr;   /* optional endpoint address */
-	void               *context;/* user-defined context */
-	uint64_t           data;    /* optional message data */
+        const struct iovec *msg_iov;  /* scatter-gather array */
+        void               **desc;    /* local request descriptors */
+        size_t             iov_count; /* # elements in iov */
+        fi_addr_t          addr;      /* optional endpoint address */
```
Comment (shefty) [inline man/fi_msg.3.md]: The formatting of this line differs from those around it.  Why was the formatting of the other lines changed?

Comment (goodell) [inline man/fi_msg.3.md]: I'm not sure why GitHub is rendering the diff like this.  I just tidied up the alignment while I was fixing the incorrect type.  That included converting to all-spaces for indentation/alignment.  If you would prefer that the alignment remain slightly jagged and partially tab-based, I can put it back.  The whitespace choices don't match the raw source in either case (it's 100% tabs with no comments in fi_endpoint.h).

Comment (shefty) [inline man/fi_msg.3.md]: I'm worried about the alignment.  On my systems, both the .md and man pages appear aligned.  There is a tab at the start of the lines, but spaces after that in the .md.  As long as things align, I don't care.

**Comment by goodell** [comment]
I just pushed an updated version that leaves the hard tabs at the front and tweaks the spacing just a bit to squeeze in better on an 80 character terminal viewing the man page.  The old version was also perfectly aligned, but GH renders the whitespace changes better in this diff.

**Pull request merged**